src/ec/packages/ec-component-file/file.html.twig

```diff
@@ -0,0 +1,112 @@
+{% spaceless %}
+{#
+  Parameters:
+  - "icon" (object) (default: {}): object of type Icon; file type
+  - "title" (string) (default: '')
+  - "language" (string) (default: '')
+  - "meta" (string) (default: '')
+  - "download" (object) (default: {}): object of type Link
+  - "translation" (array) (default: []):
+    - "toggle" (object) (default: {}): object of type Button
+    - "items" (array) (default: []):
+      - "title" (string) (default: '')
+      - "meta" (string) (default: '')
+      - "lang" (string) (default: '')
+    - "description (string) (default:'')
+  - "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+  - "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'
+#}
+
+{# Internal properties #}
+
+{% set _icon = icon|default({}) %}
+{% set _title = title|default('') %}
+{% set _language = language|default('') %}
+{% set _meta = meta|default('') %}
+{% set _download = download|default({}) %}
+{% set _translation = translation|default([]) %}
+
+{% set _css_class = 'ecl-file' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<div class="{{ _css_class }}"{{ _extra_attributes|raw }} data-ecl-file>
+  <div class="ecl-file__container">
+    {%- include '../ec-component-icon/icon.html.twig' with _icon|merge({extra_classes: 'ecl-file__icon'}) -%}
```

Comment:
No spaces between brackets, i.e. `{extra_classes: 'ecl-file__icon'}` instead of `{ extra_classes: 'ecl-file__icon' }`, it's unfortunate we don't have prettier or linter for that.

Also, could you please add a new line after `merge({`, similarly to the other implementations beneath?